src/SprykerEco/Yves/Computop/Handler/PostPlace/AbstractPostPlacePaymentHandler.php

```diff
@@ -38,6 +39,10 @@ public function handle(QuoteTransfer $quoteTransfer, array $responseArray)
     {
         /** @var \Generated\Shared\Transfer\ComputopSofortInitResponseTransfer $responseTransfer */
```

Comment:
This is bad practice. I know it's old code, but this should be eliminated.
If you know type of response, you know what's the type of converter. Please add property with proper doc block in this class.

Reply:
Moreover, I think it's not correct type hinting, it's just one of the possible types which has `header` property.

Reply:
This will require huge refactoring, please to to @alex-galych  before implementation